tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -23,7 +23,7 @@ import isDirectory from '../../../src/util/is-directory.js';
 
 function prepareExtensionRunnerParams({ params } = {}) {
   const fakeChromeInstance = {
-    process: new StubChildProcess(),
+    process: new StubChildProcess(params),
```

Comment:
If you want to more realistically mimic the result of chromium-launch, then you can assign the result to `fakeChromeInstance` instead of forwarding it through `StubChildProcess`.

Here is the type from chrome-launcher: https://github.com/GoogleChrome/chrome-launcher/blob/v1.1.1/src/chrome-launcher.ts#L85